frontend/public/components/_masthead.scss

```diff
@@ -114,6 +115,11 @@
   .co-actions-menu {
     margin-left: 0;
   }
+
+  // Override default color for .navbar-pf-vertical .nav .nav-item-iconic .fa
+  .navbar-pf-vertical .nav &__updates .nav-item-iconic .fa{
```

Comment:
style nit:

```suggestion
  .navbar-pf-vertical .nav &__updates .nav-item-iconic .fa {
```

Reply:
It might be better to just do

```css
&__updates {
  color: $color-primary !important;
}
```

At some point, increasing specificity is not any cleaner than `!important`